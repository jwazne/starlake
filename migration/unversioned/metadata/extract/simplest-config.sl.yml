connectionRef: databaseConn
jdbcSchemas:
  - schema: "db_schema"
    tableTypes:
      - "TABLE"