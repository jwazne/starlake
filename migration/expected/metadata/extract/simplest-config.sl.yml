---
version: 1
extract:
  connectionRef: databaseConn
  jdbcSchemas:
    - schema: "db_schema"
      tableTypes:
        - "TABLE"